patronload/patron.py

```diff
@@ -62,6 +67,152 @@ def populate_patron_common_fields(
     return patron_template
 
 
+def populate_staff_fields(
+    patron_template: BeautifulSoup,
+    patron_dict: dict[str, Any],
+) -> BeautifulSoup:
+    """
+    Populate the staff fields in a patron record.
+
+    The order of the patron record fields must be in same order as they in the
+    STAFF_FIELDS value list.
+
+    Args:
+        patron_template: An XML template for a patron record
+        patron_dict: A dict of patron record values.
+    """
+    if patron_dict["FULL_NAME"]:
+        split_name = patron_dict["FULL_NAME"].split(",")
+        patron_template.last_name.string = (  # type: ignore[union-attr]
+            split_name[0].strip() or ""
+        )
+        patron_template.first_name.string = (  # type: ignore[union-attr]
+            split_name[1].strip() or ""
```

Comment:
If the FULL_NAME field doesn't include a comma (unlikely, but names do not follow any rules) this will throw an IndexError.

Reply:
This has changed the logic such that either name field only gets populated if there's a comma in the full name field. Is that the intention?

Reply:
Pushed new commit